index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header id="header">
+      <nav id="nav-bar">
+        <img  id="header-img" src="images/logo_img.png">  
+        <ul>
+          <li><a class="nav-link" href="#video_container">Link 1</a></li>
+          <li><a class="nav-link" href="#video_container">Link 2</a></li>
+          <li><a class="nav-link" href="#video_container">Link 3</a></li>
+        </ul>
+      </nav>
+    </header>
+    <div id="form_container">
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+        <input id="submit" type="submit" value="Get Started" class="btn">
+      </form>
+    </div>
+    <div id="video_container">
+      <iframe id="video" width="950" height="534" src="https://www.youtube.com/embed/f9ip1iSAptQ" title="What is Microverse?" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
### class and id
- you can kindly change some id to classes to use "id" to identify a single element and "class" to group more than one.

Reply:
- [Optional] Good job @PierreBarba, As I have mentioned above, here also you could elaborate a bit here to give detail of why the changes are requested/suggested.